### DISCUSSION TITLE
Force Password change

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4846] [Q&A] [answered]
Hello Team,
Do we have an option to force the password update for User, basis on some criteria or flag?

**Comment by maduvena** [comment, marked as answer]
Hi, the key strength of the Janssen server is that it can be customized in any which way you imagine.
You can update a userclaim say `fraud_pattern = true` when some event has occurred (fraud detection event).

You can use an interception script to force re-authentication whenever fraud is detected. 
Redirect back to `/authorize` endpoint with "login=prompt" parameter. 
We have presented this in an article [here](https://github.com/JanssenProject/jans/blob/main/docs/admin/recipes/stepped-up-authn.md) - See Step no 12.

Hope this helps!
